pusher/src/Services/SocketManager.ts

```diff
@@ -121,28 +123,15 @@ export class SocketManager implements ZoneEventListener {
                 }
             })
             .on("end", () => {
-                console.warn(
-                    "Admin connection lost to back server '" +
-                        apiClient.getChannel().getTarget() +
-                        "' for room '" +
-                        roomId +
-                        "'"
-                );
+                console.warn("Admin connection lost to back server");
```

Comment:
You are removing a commit I made to add more details here. Are you sure you did not force push something not intended?